aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -631,6 +631,52 @@ void qclamp_kernel(
   });
 }
 
+void qclamp_with_tensors_kernel(
+    const Tensor& qx,
+    const Tensor& min_tensor,
+    const Tensor& max_tensor,
+    Tensor& qy) {
+  AT_DISPATCH_QINT_TYPES(qx.scalar_type(), "qclamp", [&]() {
```

Comment:
we should be able to remove the `AT_DISPATCH_QINT_TYPES` macro from these functions.  This macro helps to codegen for different values of `SCALAR_TYPE`, but in the code we are not referencing `SCALAR_TYPE`.

Reply:
Makes sense, I wasn't sure if there wasn't anything else done by this macro. Removed.